assets/js/llms-admin.js

```diff
@@ -58,7 +59,7 @@
 				multiple: false,
 				placeholder: undefined !== LLMS.l10n ? LLMS.l10n.translate( 'Select a Course/Membership' ) : 'Select a Course/Membership',
 				post_type: self.attr( 'data-post-type' ) || 'post',
-				post_statuses: self.attr( 'data-post-statuses' ) || 'publish',
+				post_statuses: self.attr( 'data-post-statuses' ) || 'publish,private',
```

Comment:
this is a bit more than allowing support for private posts only in the reporting/sales page.
I'd like a more specific approach, or if you think we should extend this default behavior we can discuss about that.

Reply:
What do you suggest?

Should I make a 'data attribute' in the `Courses` select field with all the `post_statuses`?
OR
Should I return all the default `post_statuses` of WordPress for all cases?

Reply:
> Should I make a 'data-post-statuses' attribute in the Courses select field with all the post_statuses?
isn't there already such attribute? I think you should update it with the desired status.

> all the `post_statuses`?
All the required post statuses needed to filter the sales yeah.

> Should I return all the default post_statuses of WordPress for all cases?

se above

Reply:
There are too many fields that are using `.llms-select2-post` so I chose to use the 1st option (using data attribute) for this.